src/main/proto/io/dartinc/registry/v1beta1/registry.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package io.dartinc.registry.v1beta1;
+
+option java_multiple_files = true;
+
+import "tech/figure/util/v1beta1/document.proto";
+import "tech/figure/util/v1beta1/types.proto";
+import "validate/validate.proto";
+
+/**
+Digital Asset Registry Technology (DART) ENote metadata
+ */
+message ENote {
+  Controller                                controller      = 1 [(validate.rules).message.required = true]; // Identity of the ENote controller
+  tech.figure.util.v1beta1.DocumentMetadata e_note          = 2 [(validate.rules).message.required = true]; // Metadata of the authoritative copy of the ENote
+  tech.figure.util.v1beta1.Date             signed_date     = 3 [(validate.rules).message.required = true]; // Date the ENote was signed by the borrower
+  string                                    vault_name      = 4 [(validate.rules).string.min_len = 1];      // Name of the eVault storing the Authoritative copy of the ENote
+//  bool                                      modification    = 10;                                           // Modifications to the ENote
+//  bool                                      assumption      = 20;                                           // Assumption about the ENote
```

Comment:
what are the commented-out fields for?

Reply:
I needed more feedback from our product managers and legal about those fields. Turns out they are documents so I will be updating those fields to be DocumentMetadata.